lib/ex338_web/web/coherence_web.ex

```diff
@@ -9,9 +9,9 @@ defmodule Ex338.Coherence.Web do
       # Use all HTML functionality (forms, tags, etc)
       use Phoenix.HTML
 
-      import Ex338.Router.Helpers
-      import Ex338.ErrorHelpers
-      import Ex338.Gettext
+      import Ex338Web.Router.Helpers
+      import Ex338Web.ErrorHelpers
+      import Ex338Web.Gettext
```

Comment:
Most of the time you are using the multi-alias/require/import/use syntax, but here you are using multiple single directives